CHANGES.md

```diff
@@ -1,6 +1,8 @@
 unreleased
 -------------------
 
+- Multiple errors can be reported. (#447, @burnleydev1)
```

Comment:
This description is not very informative for someone reading the changelog. It could be nice if you can explain more precisely your changes.